eisbuk-admin/src/components/atoms/CopyPasteWeekButtons/CopyPasteWeekButtons.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+
+import IconButton from "@material-ui/core/IconButton";
+import Badge from "@material-ui/core/Badge";
+
+import DeleteIcon from "@material-ui/icons/Delete";
+import FileCopyIcon from "@material-ui/icons/FileCopy";
+import AssignmentIcon from "@material-ui/icons/Assignment";
+
+const CopyPasteWeekButtons: React.FC = () => {
+  return (
+    <>
+      {/* {showWeekDeleteConfirm ? (
```

Comment:
The commented out code throughout this file can probably be deleted: the original is still present in `eisbuk-admin/src/containers/SlotsPageContainer.tsx`.

Reply:
I would leave it as is to indicate that this part is still under construction, I've just put it here so that I can use it in stories. I will finish/test it out in the next PR while creating the customer area views (and stories).